clinica/pipelines/pet_surface/pet_surface_utils.py

```diff
@@ -932,7 +939,7 @@ def get_wf(subject_id,
     else:
         tpmnii = os.path.join(os.path.expandvars('$SPM_HOME'), 'tpm', 'TPM.nii')
     if not os.path.exists(tpmnii):
-        raise IOError('Could not find TPM.nii in your SPM installation. Is your $SPM_HOME environnement variable correctly set ?')
```

Comment:
Tout ce bloc (à partir de `if 'SPMSTANDALONE_HOME' in ...`) est à remplacer par (j'avais oublié de modifier le code ici quand j'avais écrit la fonction get_tpm): 

```python 
from clinica.utils.spm import get_tpm
tpmnii = get_tpm()
```

Reply:
See faddb3e :)

Reply:
Done in faddb3e